sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -313,6 +313,14 @@ pub fn run_ft_personalize(
         log::info!("Running hash {:?}", hex::encode(hasher.clone().finalize()));
     }
 
+    let words = uds_cert_bytes
```

Comment:
nit: Can you add the same comment you added on the C side describing why this is needed?

Reply:
Oops, this was also debugging code I forgot to removed. Fixed. Thanks.